src/aosm/azext_aosm/cli_handlers/onboarding_base_handler.py

```diff
@@ -121,6 +140,19 @@ def _read_config_from_file(self, input_json_path: Path) -> dict:
 
         return config_dict
 
+    def _read_params_config_from_file(self, input_json_path) -> dict:
+        """ Reads input file, takes only the {parameters:values} + returns config as dictionary"""
+        with open(input_json_path, "r", encoding="utf-8") as _file:
+            params_schema = json.load(_file)
+
+        santised_schema = {}
+        for param in params_schema["parameters"]:
+            sanitised_param = ''.join(['_' + i.lower() if i.isupper()

```

Comment:
interesting use of `if isupper()`. 
Why do we need to keep capitalisation if there's a mix of lower and upper, but not if there's only upper case?